<?xml version="1.0"?>
<launch>
  <arg name="device" default=""/>
  <arg name="ns" default="jetracer_audio_in"/>
  <arg name="dst" default="alsasink"/>
  <arg name="do_timestamp" default="false"/>
  <arg name="format" default="mp3"/>
  <arg name="channels" default="1"/>
  <arg name="sample_rate" default="16000"/>
  <arg name="sample_format" default="S16LE"/>
  
  <group ns="$(arg ns)">
  <node name="jetracer_audio_play" pkg="audio_play" type="audio_play" output="screen">
    <param name="device" value="$(arg device)"/>
    <param name="dst" value="$(arg dst)"/>
    <param name="do_timestamp" value="$(arg do_timestamp)"/>
    <param name="format" value="$(arg format)"/>
    <param name="channels" value="$(arg channels)"/>
    <param name="sample_rate" value="$(arg sample_rate)"/>
    <param name="sample_format" value="$(arg sample_format)"/>
  </node>
  </group>
</launch>

